### ISSUE TITLE
Boxchart not visible in Clojupyter

### ISSUE THREAD

**Issue by vikeri** [metasoarous/oz#91]
Hi!

I tried using oz in Clojupyter and it works for line-charts but I can't get boxplot to work. It displays nothing. Using a very simple example:

 ```clj
(def chart
    {:data {:values [{:category "a" :quantity 1}
                     {:category "a" :quantity 2}
                     {:category "a" :quantity 1}
                     {:category "a" :quantity 4}
                     {:category "b" :quantity 3}
                     {:category "b" :quantity 4}
                     {:category "b" :quantity 2}
                     {:category "b" :quantity 5}]}
     :encoding {:x {:field "category"
                    :type "ordinal"}
                :y {:field "quantity"
                    :type "quantitative"}}
     :mark "boxplot"}
)
(oz/view! chart)
```

The same data does render a correct box-plot in vega lite editor:
[https://vega.github.io/editor/#/url/vega-lite/N4KABGBEAmCG...](https://vega.github.io/editor/#/url/vega-lite/N4KABGBEAmCGAutIC4yghSA3WAbArgKYDOKYA2uBhOtRpAMYKEDmA9gE4CeZkSANFTpQAjvlgA7eAEt4PVAEYhGAL6DhtYY2btuvAcuqQxkmXLIAmQ2DXXNdbfFad5UA8Mwmps10uG2Na0wmJ11XPkh1D2Nxb3NUABZrALp7IxDnPVRIACNIoNFYs1cAZmSo6jT6DLDePIqHL2KyJP8GmgLHTPD6zqafS3K7TpqXOvyPQtMB1ABWZOUAXSoUyEIJBjZoaQkWMntIAA9960gAM2lCXGheUb12qDkAB0JeTm2JPEhlFMxXKvOl2uvH65gekGer2yoIQ0iwrx+KyikAAtrAOABrOpsQ5PXBseDfFRAA)

The error I get in the console is, which [seems to indicate that vega-lite is old](https://github.com/altair-viz/altair/issues/1579) but looking at the cljsjs/vega-lite it seems updated. I'm using the latest version of `oz`

<img width="803" alt="Skärmavbild 2020-02-13 kl  14 13 59" src="https://user-images.githubusercontent.com/846688/74438823-45ef4f80-4e6b-11ea-8b00-ed3109d140ec.png">

**Comment by metasoarous** [comment]
Hi @vikeri. Good find! Thanks for reporting!

It looks like the code in the `oz.notebook.*` namespaces is using its own hardcoded versions, instead of the versions in `oz.core`. I'll try to get a fix for this released soon (maybe some time next week). If you're inspired to put up a PR though, that would help speed things along and earn you my eternal gratitude :-)

Thanks again!

**Comment by vikeri** [comment]
I can look into it, how do I test it offline?

**Comment by metasoarous** [comment]
If you clone, edit code, and run `lein install` then your local version _should_ get picked up. I say "should" because I don't know for sure that Clojupyter isn't doing something fancy/nonstandard regarding dependency management.

**Comment by MawiraIke** [comment]
```Clojure
...
      'vega-embed':  'vega-embed@3?noext',
      'vega-lib': 'vega-lib?noext',
      'vega-lite': 'vega-lite@2?noext',
      'vega': 'vega@3?noext'
...
```

Seems like the issue is that clojupyter uses vega-lite v2 while boxplot was added in vega-lite [v3.0.0](https://github.com/vega/vega-lite/releases/tag/v3.0.0). Works after changing the versions. 

![working](https://user-images.githubusercontent.com/38689046/76092341-4beed100-5fd0-11ea-8814-061d45a6a1d6.png)

Making a PR from my fork to fix this.

**Comment by metasoarous** [comment]
Thanks @MawiraIke!

If you would, please use the version vars in `oz.core` (`vega-version`, `vega-lite-version`, etc) so that we avoid this problem in the future and keep all versions in sync.

Thanks again!

**Comment by MawiraIke** [comment]
Ok sure, fixed it on this commit 227ce918e1e8b36926c0a10d428bf59d9f65ffab.

**Comment by metasoarous** [comment]
Seems like we're good to go here then. Thanks again @MawiraIke!
